aviary/interface/methods_for_level2.py

```diff
@@ -1604,8 +1604,10 @@ def add_driver(self, optimizer=None, use_coloring=None, max_iter=50, verbosity=V
         if verbosity != Verbosity.QUIET:
             if isinstance(verbosity, list):
                 driver.options['debug_print'] = verbosity
-            elif verbosity.value >= 2:
+            elif verbosity.value >= 3:
                 driver.options['debug_print'] = ['desvars', 'ln_cons', 'nl_cons', 'objs']
+        if verbosity is not Verbosity.DEBUG and optimizer in ("SNOPT", "IPOPT"):
```

Comment:
Hang on, I think we want the pyoptsparse resulst for for verbose. and above. 

I would make it "if verbosity < Verbosity.VERBOSE"

Reply:
I responded on the teams, since I didn't see a way to add a collapsible comment here